src/main/java/deez/TaskList.java

```diff
@@ -0,0 +1,105 @@
+package deez;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+import deez.tasks.Task;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList implements Serializable {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Names of `final` constants should be in SCREAMING_SNAKE_CASE, as per CS2103T code quality guidelines 😄

Reply:
Hi! Actually, the reason why this isn't in screaming snake case is because Java's `Serializable` interface enforces a `serialVersionUID` property that is named in a way that does not follow the CS2103T code standards.

[See info about Serializable here](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/io/Serializable.html). Without a `private static final long` named `serialVersionUID`, the compiler doesn't allow `TaskList` to be `Serializable` 😅

I'm not sure if there is a way around this but I'm open to suggestions!